addons/pharma/XEH_preInit.sqf

```diff
@@ -252,6 +252,53 @@ PREP_RECOMPILE_END;
     true
 ] call CBA_Settings_fnc_init;
 
+// Pervitin Settings Category
+
+[
+    QGVAR(medLvl_Pervitin),
+    "LIST",
+    [LLSTRING(medLvl_Pervitin)],
+    [CBA_SETTINGS_CAT, LSTRING(SubCategory_Pervitin)],
+    [[0, 1, 2], ["STR_ACE_Medical_Treatment_Anyone", "STR_ACE_Medical_Treatment_Medics", "STR_ACE_Medical_Treatment_Doctors"], 0],
```

Comment:
[ACECSTRING (medical_treatment, Anyone), ACECSTRING(medical_treatment, Medics), ACECSTRING(medical_treatment, Doctors)]

Reply:
here it should be ACELSTRING